### ISSUE TITLE
About validation set in training pharse

### ISSUE THREAD

**Issue by PhanVinhLong** [open-mmlab/mmdetection#4057]
Does validation set play a role in training parameters for model? Or it's just used to choose hyperparameters (e.g numer of epochs...).
Thank you.

**Comment by ZwwWayne** [comment]
Yes, you can do an ablation study of hyper-parameters using the validation set.
